core/app/models/spree/taxon/paperclip_attachment.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Spree::Taxon::PaperclipAttachment
+  extend ActiveSupport::Concern
+
+  included do
+    has_attached_file :icon,
+      styles: { mini: '32x32>', normal: '128x128>' },
+      default_style: :mini,
+      url: '/spree/taxons/:id/:style/:basename.:extension',
+      path: ':rails_root/public/spree/taxons/:id/:style/:basename.:extension',
+      default_url: '/assets/default_taxon.png'
+
+    validates_attachment :icon,
+      content_type: { content_type: ["image/jpg", "image/jpeg", "image/png", "image/gif"] }
```

Comment:
Under `core/app/models/spree/image/paperclip_attachment.rb`, we have `%w(image/jpeg image/jpg image/png image/gif)` —I think, for consistency, we should use the same syntax in here too